src/PrimeTower.cpp

```diff
@@ -51,7 +51,7 @@ void PrimeTower::computePrimeTowerMax(SliceDataStorage& storage)
         
     extruder_count = storage.getSettingAsCount("machine_extruder_count");
     
-    int max_object_height_per_extruder[extruder_count]; 
+    int *max_object_height_per_extruder = (int*)alloca(sizeof(int)*extruder_count); 
```

Comment:
This function has been refactored. It's now moved to `src/FffPolygonGenerator.cpp`, named as `FffPolygonGenerator::computePrintHeightStatistics`, and it's not necessary to have this change any more.